chart/templates/clusterrole.yaml

```diff
@@ -149,3 +149,72 @@ rules:
       - get
       - watch
       - patch
+  # Need to create Flux resources on the seed for Flux managed addons
+  - apiGroups:
+    - notification.toolkit.fluxcd.io
+    resources:
+      - alerts
+      - providers
+      - receivers
+    verbs:
+      - get
+      - list
+      - watch
+      - patch
+      - update
+      - create
+      - delete
+  - apiGroups:
+    - source.toolkit.fluxcd.io
+    resources:
+    - buckets
+    - gitrepositories
+    - helmcharts
+    - ocirepositories
+    - helmrepositories
+    verbs:
+      - get
+      - list
+      - watch
+      - patch
+      - update
+      - create
+      - delete
+  - apiGroups:
+    - helm.toolkit.fluxcd.io
+    resources:
+    - helmreleases
+    verbs:
+      - get
+      - list
+      - watch
+      - patch
+      - update
+      - create
+      - delete
+  - apiGroups:
+    - kustomize.toolkit.fluxcd.io
+    resources:
+    - kustomizations
```

Comment:
For kustomizations and helmreleases, I think we should monitor them using this pattern for the stackhpc add ons:
https://github.com/azimuth-cloud/azimuth-capi-operator/blob/564bb0b6ad1a7d4a1189cbbda94cc644ed48c9fb/azimuth_capi/operator.py#L699